lib/pbench/cli/agent/commands/results/move.py

```diff
@@ -104,9 +104,11 @@ def execute(self, single_threaded: bool, delete: bool = True) -> int:
                         self.config,
                         self.logger,
                     )
-                    crt.copy_result_tb(
+                    res = crt.copy_result_tb(
                         self.context.token, self.context.access, self.context.metadata
                     )
+                    if not res.ok:
+                        raise CopyResultTb.FileUploadError
```

Comment:
This code is raising the _class_ `FileUploadError`; it should be raising an _instance_ of it.  See [example](https://github.com/distributed-system-analysis/pbench/blob/e2cbf4b6cc873dee0fc6e307fb0162c74184bbc0/lib/pbench/agent/results.py#L377).

Reply:
Fixed.

Reply:
Fixed.